lib/drand.ts

```diff
@@ -0,0 +1,69 @@
+import HTTP, {ClientOptions, Watcher} from './http'
+import VerifyingClient from './verifying-client'
+import OptimizingClient from './optimizing-client'
+
+export type DrandOptions = Partial<{
+    chainHash: string
+    insecure: boolean
+    disableBeaconVerification: boolean
+    chainInfo: ChainInfo
+    cacheSize: number
+    watcher: Watcher
+}>
+
+export interface NetworkClient {
+    get(round?: number, options?: ClientOptions): Promise<RandomnessBeacon>
```

Comment:
Should we had the chainhash as an optional param here rather as in the DrandOptions?